src/v2/lib/entity-handlers.js

```diff
@@ -36,6 +35,12 @@ const getFunctionName = (action, entityKey) => {
   return `${action}${startCase(entityKey)}`.replace(/\s/g, '')
 }
 
+function startCase (key) {
+  const words = key.split(' ')
+  return words.map((word) => {
+    return word[0].toUpperCase() + word.substring(1)
+  }).join(' ')
+}
```

Comment:
Tiny formatting tweak: add an empty line between this function and the comment for the next function.

```suggestion
}

```